src/main/resources/liquibase/202003261146-add-table-for-account-config.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202003261146-1" author="mstead">
+        <comment>Add table for account configuration for tally/conduit</comment>
+        <createTable tableName="account_config">
+            <column name="account_number" type="VARCHAR(32)" />
+            <column name="sync_enabled" type="BOOLEAN">
+                <constraints nullable="false" />
+            </column>
+            <column name="reporting_enabled" type="BOOLEAN">
+                <constraints nullable="false" />
+            </column>
+            <column name="opt_in_type" type="VARCHAR(12)">
```

Comment:
In case we add more values, it's better to make this longer (`varchar(255)`). Interestingly, I've been doing [some reading](https://dba.stackexchange.com/a/21496) that suggests just using type `text` is just as performant (and more future proof, since length is then unlimited) - and [implemented the same under the hood](https://stackoverflow.com/a/4849030). So I'd be fine with with either `varchar(255)` or `text` here.

Reply:
DONE - Went with varchar(255) since TEXT was causing issues with primary-keys. Decided to keep it consistent.